sync/class.jetpack-sync-module-themes.php

```diff
@@ -414,9 +431,34 @@ private function get_theme_support_info() {
 		}
 
 		$theme = wp_get_theme();
-		$theme_support['name'] = $theme->name;
-		$theme_support['version'] =  $theme->version;
+		$theme_support['name'] = $theme->get('Name');
+		$theme_support['version'] =  $theme->get('Version');
+		$theme_support['slug'] = $theme->get_stylesheet();
+		$theme_support['uri'] = $theme->get('ThemeURI');
+
 
 		return $theme_support;
 	}
+
+	private function get_delete_theme_call() {
```

Comment:
that would have been my preference, but one of the values returned by that method is used https://github.com/Automattic/jetpack/pull/7555/files/0bc9c9158d7fbfc1172ad739e52979ff48a21184#diff-37af365e9198e952a002bbf543cdcdcfR151